feat: Post-purchase page (#67)
octopusthink/getmicdrop.com: src/content/pages/thank-you.mdx

---
title: Thank you!
---

import DownloadLink from 'components/DownloadLink';
import Panel from 'components/Panel';

<Panel>

Thank you so much for buying Mic Drop! ❤️ Your purchase supports independent software developers and helps fund new features. 

## Getting your license

> You should receive a receipt and a license key via email momentarily. If it doesn't come through (check your spam folders just in case!), please [drop us a line](mailto:help@getmicdrop.com) and we'll send it over manually. 

Reviewer: ```suggestion
You should receive a receipt and a license key via email momentarily. If it doesn't come through (check your spam folders just in case!), please [drop us a line](mailto:help@getmicdrop.com) and we'll get you sorted. 
```

## How to activate your license

> 1. <DownloadLink>Download Mic Drop</DownloadLink> and install it, if you haven't already.

Reviewer: ```suggestion
1. <DownloadLink source="post-purchase">Download Mic Drop</DownloadLink>, if you haven't already.
```

2. Open Mic Drop and click on the mic icon in the menu bar to open the menu.
3. Select "License" from the menu.

> 4. Enter your license key in the screen provided.

Reviewer: ```suggestion
4. Enter your email and license key in the screen provided.
```
